### ISSUE TITLE
[Bug] Text color menu is hard to see.

### ISSUE THREAD

**Issue by Joelute** [AppFlowy-IO/AppFlowy#6762]
### Bug Description

When you attempt to change the text color and open the color menu, the menu itself will be initialized without any borders to help you differentiate it from the background.

### How to Reproduce

1. Highlight any text
2. Click on text color to open up the menu

### Expected Behavior

The menu should open up with its borders just like other menus.

### Operating System

Arch Linux 

### AppFlowy Version(s)

0.7.3

### Screenshots

![image](https://github.com/user-attachments/assets/3c254df3-1d3e-4dfc-96de-8da531b97873)

### Additional Context

If this is an reproducible issue, I would be happy to investigate it.

**Comment by Joelute** [comment]
Hey @LucasXu0, I noticed that you've self assigned yourself to this issue. Would it be possible if I could take over the issue instead? I would love to take a jab at this :)

**Comment by LucasXu0** [comment]
@Joelute Of course. Let me know if you need any help. In the meantime, I'll look into why this happened and share my findings here.

**Comment by Joelute** [comment]
I just got my dev environment setup. Found out that this also happens to highlight color if the selected text isn't highlighted.
![image](https://github.com/user-attachments/assets/c79ff35f-0091-4a15-9706-42b95d42edca)
